src/components/Home.js

```diff
@@ -0,0 +1,22 @@
+import { useSelector } from 'react-redux';
+import Layout from './layout/Layout';
+
+export default function Home() {
+  const {
+    neighborsMain, country, city, iso,
+  } = useSelector((state) => state.location);
+
+  return (country
+    && (
+      <Layout
+        title="NEIGHBORING COUNTRIES"
+        neighbors={neighborsMain}
+        country={country}
+        city={city}
+        iso={iso}
+      >
+        <h1><span>{`${country.toUpperCase()}, ${city.toUpperCase()}`}</span></h1>
+      </Layout>
+    )
+  );
+}
```

Comment:
Your Home page should have a list of categories that could be filtered using given parameters, e.g. category name. Please try and add this feature so that your project meets the requirements. You could filtering by alphabetical order the names of the neighboring countries 🤓

Reply:
Done